public/app/plugins/panel/graph3/migrations.ts

```diff
@@ -0,0 +1,266 @@
+  }
+
+  // "seriesOverrides": [
+  //   {
+  //     "$$hashKey": "object:183",
+  //     "alias": "B-series",
+  //     "fill": 3,
+  //     "nullPointMode": "null as zero",
+  //     "lines": true,
+  //     "linewidth": 2
+  //   }
+  // ],
+  if (angular.seriesOverrides?.length) {
+    for (const seriesOverride of angular.seriesOverrides) {
+      if (!seriesOverride.alias) {
+        continue; // the matcher config
+      }
+      const rule: ConfigOverrideRule = {
+        matcher: {
+          id: FieldMatcherID.byName,
+          options: seriesOverride.alias,
+        },
+        properties: [],
+      };
+      for (const p of Object.keys(seriesOverride)) {
+        const v = seriesOverride[p];
+        switch (p) {
+          // Ignore
+          case 'alias':
+          case '$$hashKey':
+            break;
+          // Link to y axis settings
+          case 'yaxis':
+            if (2 === v) {
+              const y2 = getFieldConfigFromOldAxis(yaxes[1]);
+              fillY2DynamicValues(y1, y2, rule.properties);
+            }
+            break;
+          case 'fill':
+            rule.properties.push({
+              id: 'custom.fillOpacity',
+              value: v / 10.0, // was 0-10
+            });
+            break;
+          case 'points':
+            rule.properties.push({
+              id: 'custom.showPoints',
+              value: v ? PointVisibility.Always : PointVisibility.Never,
+            });
+            break;
+          case 'bars':
+            if (v) {
+              rule.properties.push({
+                id: 'custom.drawStyle',
+                value: DrawStyle.Bars,
+              });
+              rule.properties.push({
+                id: 'custom.fillOpacity',
+                value: 1, // solid bars
+              });
+            } else {
+              rule.properties.push({
+                id: 'custom.drawStyle',
+                value: DrawStyle.Line, // Change from bars
+              });
+            }
+            break;
+          case 'lineWidth':
+            rule.properties.push({
+              id: 'custom.lineWidth',
+              value: v,
+            });
+            break;
+          case 'pointradius':
+            rule.properties.push({
+              id: 'custom.pointSize',
+              value: v,
+            });
+            break;
+          default:
+            console.log('Ignore override migration:', seriesOverride.alias, p, v);
+        }
+      }
+      if (rule.properties.length) {
+        overrides.push(rule);
+      }
+    }
+  }
+
+  const graph = y1.custom ?? ({} as GraphFieldConfig);
+  graph.drawStyle = angular.bars ? DrawStyle.Bars : angular.lines ? DrawStyle.Line : DrawStyle.Points;
+  if (angular.points) {
+    graph.showPoints = PointVisibility.Always;
+  } else if (graph.drawStyle !== DrawStyle.Points) {
+    graph.showPoints = PointVisibility.Never;
+  }
+  if (graph.drawStyle === DrawStyle.Bars) {
+    graph.fillOpacity = 1.0; // bars were always
```

Comment:
when i do a single-series bars style in Flot and disable lines, then switch to uPlot, it seems to reach here, but the bars opacity does not end up with 1.0, but the default 0.1. additionally, the bars have a non-zero lineWidth (i would expect this to be set to zero when migrating from flot).